cabal.GHC-8.8.4.config

```diff
@@ -282,9 +282,9 @@ constraints: any.Cabal ==3.0.1.0,
              any.syb ==0.7.2.1,
              any.tagged ==0.8.6.1,
              tagged +deepseq +transformers,
-             any.tasty ==1.4.1,
+             any.tasty ==1.2.3,
```

Comment:
It's a bit curious that this version number _decreased_. Do you have a sense for why?

Similar comments apply to the version number for `tasty-ant-xml` below.

Reply:
It's due to the inclusion of `--enable-tests` in the cabal configuration. The constraint solver needed to bump the version of tasty down to satisfy a test in `aig`. We are not running those tests, but it's not currently possible to tell the constraint solver that ):

Reply:
I see! Perhaps I should just just submit a PR to `aig`, then :)

Reply:
See GaloisInc/aig#6.